content/learn/development-guides/analysis-applications.md

```diff
@@ -1,79 +1,73 @@
 
-**Consumer Vignettes** These vignettes highlight the process of certain programmatic tasks that involve the DCP.
-
-[Consumer Vignettes](/learn/development-guides/consumer-vignettes)
-
-## Working with Ingest
-
-**Ingest API** The Ingest Service provides a programmatic API for submitting data. This may be useful, for example, for LIMs integration solutions. This does require authentication and would require contacting the ingest team at [data-help@humancellatlas.org](mailto:data-help@humancellatlas.org).
-
-[Starting point: GitHub Repository Ingest Central (ingest-central)](https://github.com/HumanCellAtlas/ingest-central)    
-[GitHub Repository for Ingest API(ingest-broker-api)](https://github.com/HumanCellAtlas/ingest-broker-api)    
+[Ingest Broker Development Guide](https://prod.data.humancellatlas.org/learn/development-guides/ingest-broker-development-guide)   
+[GitHub Repository: Ingest Central (ingest-central)](https://github.com/HumanCellAtlas/ingest-central) (a good starting point)   
+[GitHub Repository: Ingest API (ingest-broker-api)](https://github.com/HumanCellAtlas/ingest-broker-api)   
 [HAL Browser for the Ingest API](http://api.ingest.dev.data.humancellatlas.org/browser/index.html)   
 
-## Learning More About Metadata
+## Learning about Metadata
 
-**HCA Metadata** The [HCA metadata schema](https://dev.data.humancellatlas.org/learn/metadata/structure) is a curated description of the entities and thier relationships captured in HCA data. The metadata schema is public for review on GitHub. The metadata schema is also extensible and open for changes through pull requests.
+The [HCA metadata standard](https://dev.data.humancellatlas.org/learn/metadata/structure) is the common set of guidelines the HCA uses to describe samples, protocols, and files associated with projects. The metadata standard is expressed as JSON schemas representing the submitted entities, their associated fields, and the relationships between them. The HCA is committed to evolving the metadata standards based on the needs of the scientific community, so everyone is welcome to suggest metadata updates by creating GitHub issues or emailing the metadata team at [data-help@humancellatlas.org](mailto:data-help@humancellatlas.org).
 
-[Learn More About Metadata](https://github.com/HumanCellAtlas/metadata-schema/tree/master/docs)   
-[Github Repository for metadata schema (metadata-schema)](https://github.com/HumanCellAtlas/metadata-schema)   
-[GitHub Repository for the metadata API Library (metadata-api)](https://github.com/HumanCellAtlas/metadata-api)   
+[Metadata Dictionary](https://dev.data.humancellatlas.org/learn/metadata/metadata-dictionary)   
```

Comment:
change to internal link structure: /learn/metadata/metadata-dictionary

Reply:
Done!